pystac_client/collection_client.py

```diff
@@ -6,13 +6,14 @@
 from pystac_client.conformance import ConformanceClasses
 from pystac_client.exceptions import APIError
 from pystac_client.item_search import ItemSearch
+from pystac_client.mixins import QueryablesMixin
 from pystac_client.stac_api_io import StacApiIO
 
 if TYPE_CHECKING:
     from pystac.item import Item as Item_Type
 
 
-class CollectionClient(pystac.Collection):
+class CollectionClient(pystac.Collection, QueryablesMixin):
     modifier: Callable[[Modifiable], None]
     _stac_io: Optional[StacApiIO]
```

Comment:
Since we define the `_stac_io` attribute on `StacAPIObject`, do we need to redefine it here?

Reply:
Oh good point!